components/modal/modal.js

```diff
@@ -256,6 +258,10 @@ Modal.propTypes = {
    * Determine whether a overlay is visible or not
    */
   overlay: PropTypes.bool,
+  /**
+   * Modal size
+   */
+  size: PropTypes.oneOf(['small', 'big']),
```

Comment:
Since we have only 2 possible choices is it better to go with `true/false` variant? In that case we won't have unused class like - `size_big`